zio-telemetry-example-proxy/src/main/scala/zio/telemetry/example/proxy/http/StatusesResponse.scala

```diff
@@ -0,0 +1,5 @@
+package zio.telemetry.example.proxy.http
+
+final case class StatusResponse(name: String, version: String, status: String)
+
+final case class StatusesResponse(statuses: StatusResponse)
```

Comment:
Based exclusively on naming, I'd presume `statuses` should be `List[StatusResponse]`. I'd probably do the following as well:

- Make `StatusesResponse` value class.
- Drop `Response` suffix.
- Add encoders to their companions.